src/commonMain/kotlin/com/algolia/search/model/response/ResponseHitsWithPosition.kt

```diff
@@ -0,0 +1,11 @@
+package com.algolia.search.model.response
+
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+public data class ResponseHitsWithPosition(
```

Comment:
Single hit => singular :upside_down_face: 

```suggestion
public data class ResponseHitWithPosition(
```